src/DotNetWorker/FunctionsApplication.cs

```diff
@@ -40,20 +38,18 @@ public FunctionContext CreateContext(IInvocationFeatures features)
             return _functionContextFactory.Create(features);
         }
 
-        public Task<WorkerInitResponse> InitializeWorkerAsync(WorkerInitRequest request)
+        public Task<InitializationResponse> InitializeAsync()
         {
-            var response = new WorkerInitResponse()
+            var response = new InitializationResponse
             {
-                Result = new StatusResult { Status = Status.Success }
+                WorkerVersion = typeof(FunctionsApplication).Assembly.GetName().Version?.ToString()
             };
 
             response.Capabilities.Add("RpcHttpBodyOnly", bool.TrueString);
             response.Capabilities.Add("RawHttpBodyBytes", bool.TrueString);
             response.Capabilities.Add("RpcHttpTriggerMetadataRemoved", bool.TrueString);
             response.Capabilities.Add("UseNullableValueDictionaryForHttp", bool.TrueString);
 
```

Comment:
All should be moved to GrpcWorker? Maybe instead of "InitializeAsync" there could be a WorkerVersion property that's used by GrpcWorker?

Reply:
That's a great point -- these capabilities are specifically for Grpc. I think moving this all to the worker would make sense (and not even exposing the WorkerVersion property anywhere... that seems like a Grpc implementation detail as well).

Reply:
Done.